deploy/crds/operator.ibm.com_operandbindinfos_crd.yaml

```diff
@@ -92,8 +92,11 @@ spec:
               type: array
           type: object
       type: object
-  version: v1alpha1
+  version: v1
```

Comment:
bindinfo is newly added API, you don't need to keep `v1alpha1`, just use `v1`.

Reply:
This is auto-generated by `operator-sdk generate crds`, because the bindinfo existing into the API v1alpha1, if delete bindinfo API from v1alpha1, this CRD will only one version